Documentation/migration-guide.md

```diff
@@ -0,0 +1,244 @@
++ implementation 'com.adobe.marketing.mobile:edge:2.+'
++ implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
++ implementation 'com.adobe.marketing.mobile:edgemedia:2.+'
+```
+
+> **Warning**  
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing Gradle dependencies.
+
+------
+
+### Import and register extensions
+
+##### Java
+
+```diff
+import com.adobe.marketing.mobile.MobileCore;
+- import com.adobe.marketing.mobile.Identity;
+- import com.adobe.marketing.mobile.Analytics;
+- import com.adobe.marketing.mobile.Media;
++ import com.adobe.marketing.mobile.Edge;
++ import com.adobe.marketing.mobile.edge.identity.Identity;
++ import com.adobe.marketing.mobile.edge.identity.Media;
+```
+
+##### Java
+
+```diff
+public class MainApp extends Application {
+    private final String ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID";
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        MobileCore.setApplication(this);
+        MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+
+-        List<Class<? extends Extension>> extensions = Arrays.asList(
+-                Media.EXTENSION, Analytics.EXTENSION, Identity.EXTENSION);
++       List<Class<? extends Extension>> extensions = Arrays.asList(
++                Media.EXTENSION, Edge.EXTENSION, Identity.EXTENSION);
+        MobileCore.registerExtensions(extensions, o -> {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized");
+        });
+    }
+}
+```
+
+
+##### Kotlin
+
+```diff
+import com.adobe.marketing.mobile.MobileCore
+- import com.adobe.marketing.mobile.Identity
+- import com.adobe.marketing.mobile.Analytics
+- import com.adobe.marketing.mobile.Media
++ import com.adobe.marketing.mobile.Edge
++ import com.adobe.marketing.mobile.edge.identity.Identity
++ import com.adobe.marketing.mobile.edge.identity.Media
+```
+
+##### Kotlin
+
+```diff
+class MyApp : Application() {
+    val ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID"
+    
+    override fun onCreate() {
+        super.onCreate()
+        MobileCore.setApplication(this)
+        MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID)
+
+-        val extensions = listOf(Media.EXTENSION, Analytics.EXTENSION, Identity.EXTENSION)
++        val extensions = listOf(Media.EXTENSION, Edge.EXTENSION, Identity.EXTENSION)
+        MobileCore.registerExtensions(extensions) {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized")
+        }
+    }
+}
+```
+
+<details>
+  <summary>Using both Media and EdgeMedia for a side-by-side comparison?</summary>
+  </br>
+  <p>If you wish to use both the extensions together during migration time for a side-by-side comparison, use the Java package name along with the extension class names for registration, as well as for any classes that use APIs from both the modules.</p>
```

Comment:
Suggest rewording: "Both Media and Media for Edge Network extensions may be used in the same application if needed. Use the Java package name when referring to the extension class or public APIs to differentiate between the two extensions."